public/img/moz-logo-bw-rgb.svg

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We should probably run this through svgo, but probably not worth holding up the PR.